docs/source/_templates/layout.html

```diff
@@ -61,7 +61,7 @@
 	  if (downloadNote.length >= 1) {
 	      var tutorialUrl = $("#tutorial-type").text();
 	      var githubLink = "https://github.com/pytorch/rl/blob/main/tutorials/sphinx-tutorials/"  + tutorialUrl + ".py",
-		  notebookLink = $(".reference.download")[1].href,
+		  notebookLink = $(".reference.download")[0].href,
```

Comment:
The issue was that the ".reference.download" selects both the `.py` and `.ipynb` links, and the `.ipynb` one happens to appear first in the html that sphinx-gallery generates. But I'm not sure what determines the order or whether it could change between different versions of sphinx-gallery

Reply:
Ah, there is an issue/PR in sphinx-gallery where "sphx-glr-download-jupyter" was added for public use: https://github.com/sphinx-gallery/sphinx-gallery/issues/622

So it seems like it should be safe to use it